contrib/gitian-keys/keys.txt

```diff
@@ -0,0 +1,28 @@
+152812300785C96444D3334D17565732E08E5E41 Andrew Chow
+E944AE667CF960B1004BC32FCA662BE18B877A60 Andreas Schildbach
+07DF3E57A548CCFB7530709189BBB8663E2E65CE Matt Corallo (BlueMatt)
+912FD3228387123DC97E0E57D5566241A0295FA9 BtcDrak
+C519EBCF3B926298946783EFF6430754120EC2F4 Christian Decker (cdecker)
+F20F56EF6A067F70E8A5C99FFF95FAA971697405 centaur
+C060A6635913D98A3587D7DB1C2491FFEB0EF770 Cory Fields
+BF6273FAEF7CC0BA1F562E50989F6B3048A116B5 Dev Random
+9A1689B60D1B3CCE9262307A2F40A9BF167FBA47 Erik Mossberg (erkmos)
+E777299FC265DD04793070EB944D35F9AC3DB76A Michael Ford
+01CDF4627A3B88AAE4A571C87588242FBE38D3A8 Gavin Andresen
+D3CC177286005BB8FF673294C5242A1AB3936517 jl2012
+D2D1085B9425F9DEFA06E7932270E30C522739F6 Jonas Schnelli
```

Comment:
This key "D2D1085B9425F9DEFA06E7932270E30C522739F6" has expired, and I couldn't look it up. We could probably remove it given @jonasschnelli has another key listed.